services/management/ia/templates_service.go

```diff
@@ -307,27 +427,50 @@ func (svc *TemplatesService) ListTemplates(ctx context.Context, req *iav1beta1.L
 			Severity:    managementpb.Severity(r.Severity),
 			Labels:      r.Labels,
 			Annotations: r.Annotations,
-			Source:      iav1beta1.TemplateSource_TEMPLATE_SOURCE_INVALID, // TODO
+			Source:      r.Source,
```

Comment:
nit: i guess this loop can be extracted out of the method, its getting too big, maybe move it to a `makeTemplate` function, something like that

Reply:
Well, I thought about that. In fact, this loop is almost everything that this method does, so extracting it will just produce a big function. Also, this code is not reusable now. IMHO better to leave it as is.